x/bandwidth/model/bandwidth.go

```diff
@@ -0,0 +1,20 @@
+package model
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	linotypes "github.com/lino-network/lino/types"
+)
+
+// BandwidthInfo - stores info about the moving average of mps and max mps
+type BandwidthInfo struct {
+	GeneralMsgEMA sdk.Dec `json:"general_msg_ema"`
+	AppMsgEMA     sdk.Dec `json:"app_msg_ema"`
+	MaxMPS        sdk.Dec `json:"max_mps"`
+}
+
+// BlockInfo - stores info about number of tx in last block
+type BlockInfo struct {
+	TotalMsgSignedByApp  uint32         `json:"total_tx_signed_by_app"`
```

Comment:
suggest to use `int64` instead of unsigned type. Because if the intermediate result of unsigned types is negative, it will overflow, which sometimes is hard to avoid and easy to forget.